config/locales/de.yml

```diff
@@ -0,0 +1,204 @@
+
+  dashboards:
+    show:
+      latest: "Letzte Aktualisierungen"
+      latest_title: "Neueste Aktualisierungen RSS Feed"
+      no_subscriptions: "Du hast noch keine Gems abonniert. Besuche %{gem_link} um ein Gem zu abonnieren!"
+      gem_link_text: "Gem-Seite"
+      mine: "Meine Gems"
+      no_owned: "Du hast noch keine Gems gepusht. Schau doch vielleicht die Dokumentation auf %{creating_link} an und %{migrating_link} ein Gem from RubyForge."
+      creating_link_text: 'Erstellen'
+      migrating_link_text: 'Migrieren'
+      my_subscriptions: 'Abonnements'
+
+  home:
+    index:
+      downloads_counting: 'Gezählte Downloads'
+      find_blurb: "Finde, installiere und veröffentliche RubyGems."
+      learn:
+        install_rubygems: "Installiere RubyGems"
+
+  layouts:
+    application:
+      header:
+        dashboard: "Dashboard"
+        sign_in: "Anmelden"
+        sign_up: "Registrieren"
+        sign_out: "Abmelden"
+        search_gem: "Suche Gems&hellip;"
+      footer:
+        status: "Status"
+        uptime: "Betriebszeit"
+        source_code: "Code"
+        data: "Daten"
+        discussion_forum: "Diskussion"
+        statistics: "Statistiken"
+        blog: "Blog"
+        about: "Über uns"
+        help: "Hilfe"
+        guides: "Dokumentation"
+        contribute: "Beitragen"
+        supported_by: "Unterstützt von"
+        hosted_by: "Gehostet von"
+        designed_by: "Design von"
+        resolved_with: "Aufgelöst mit"
+        optimized_by: "Optimiert mit"
+        tracking_by: "Getrackt von"
+        monitored_by: "Überwacht von"
+        gems_served_by: "Gems serviert von"
+
+  pages:
+    about:
+      title: Über uns
+      purpose:
+        header: "Willkommen in %{site}, die Ruby-Community für Gem-Hostingservice. Die Gründe sind vielfältig:"
+        better_api: Bereitstellen eines besseren APIs für den Umgang mit Gems
+        transparent_pages: Erstellen transparenter und zugänglicher Projektseiten
+        enable_community: Der Gemeinschaft ermöglichen, die Webseite zu verbessern und zu erweitern
+      founding: "Das Projekt wurde im April 2009 von %{founder} gestartet und hat seitdem über %{contributors} und %{downloads}. Ab dem Release von RubyGems 1.3.6 wurde die Website von Gemcutter zu %{title} umbenannt, um eine zentrale Rolle in der Ruby-Community zu verfestigen."
+      support: "Obwohl RubyGems.org nicht von einem bestimmten Unternehmen ausgeführt wird, haben schon viele uns bisher geholfen. Das aktuelle Design, die Illustrationen und die Front-End-Entwicklung für dieser Webseite wurden von %{dockyard} erstellt. %{github} hat auch von unschätzbarem Wert uns geholfen den Code mit anderen Entwicklern zuteilen, um an dem Code leichter zusammenarbeiten. Die Website wurde mit %{heroku} gestartet, dessen großen Dienst uns geholfen, RubyGems als eine praktikable Lösung zu erweisen, wo die ganze Gemeinschaft darauf verlassen kann."
```

Comment:
I would change "haben schon viele uns bisher geholfen" to "haben viele Unternehmen uns bereits unterstützt"